lib/services/elasticsearch.js

```diff
@@ -994,8 +994,8 @@ function getElasticsearchRequest(request, kuzzle, extraParams = []) {
   const data = {};
 
   if (request.input.resource.index) {
-    if (request.input.resource.index === kuzzle.internalEngine.index) {
-      throw new BadRequestError(`Cannot operate on Kuzzle internal index "${kuzzle.internalEngine.index}"`);
+    if (request.input.resource.index[0] === '%') {
```

Comment:
These changes make the `kuzzle` argument unused. It should be removed from this function's prototype.